pkgs/applications/misc/clipqr/default.nix

```diff
@@ -0,0 +1,54 @@
+{ buildGoModule, copyDesktopItems, fetchFromGitLab, lib, makeDesktopItem, pkgs, pkgconfig, ... }:
+
+buildGoModule rec {
+  pname = "clipqr";
+  version = "1.0.0";
+
+  src = fetchFromGitLab {
+    owner = "imatt-foss";
+    repo = "clipqr";
+    rev = "v${version}";
+    sha256 = "sha256-E90nTJtx4GOacu8M7oQBznnSQVDIZatibgKMZEpTUaQ=";
+  };
+
+  vendorSha256 = "5kAOSyVbvot4TX/XfRNe1/zZk6fa7pS1Dvn9nz11u3U=";
+
+  buildInputs = with pkgs; [
+    libGL
+    mesa
+    xorg.libX11.dev
+    xorg.libXcursor
+    xorg.libXext.dev
+    xorg.libXi
+    xorg.libXinerama
+    xorg.libXrandr
```

Comment:
Each X11 library should imported as an input separately, i.e, `[ libX11 libXcurosor libXext ]`. Also, I think there's no reason to explicitly specify the `dev` output.

Reply:
Sorry I don't understand how I should import x11 libs as an separate input, I looked at others projects but didn't find an example that does this. Could you please give a hint? Thanks again for all help :)

Reply:
Something like this:
```nix
{ lib
, fetchFromGitHub
, ....... etc
, libX11
, libXcursor
, ..... etc
}: ...... etc

stdenv.mkDerivation rec {
..... etc
  buildInputs = [
    libX11
    libXcursor
  ]; 
```

Reply:
Ahah ok I got it, sorry I'm pretty new to Nix. Thanks again, you are awesome :)